.github/pull_request_template.md

```diff
@@ -1,12 +1,13 @@
 ### Description
 
-
 ### Dragons
 
 Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
 
-
 ### How Has This Been Tested?
 
 Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
 
+### Is any new monitoring/alerting being added along with this change?
```

Comment:
how about just

`### How will this change be monitored?`

Reply:
Much better! Done